polaris-react/src/components/LegacyFilters/LegacyFilters.scss

```diff
@@ -0,0 +1,227 @@
+@import '../../styles/common';
+
+$list-filters-header-height: $top-bar-height;
+$list-filters-footer-height: 70px;
+
+.Filters {
```

Comment:
For legacy component migrations for LegacyStack and LegacyCard, we updated these class names to include `Legacy` in them, so it would be `.LegacyFilters`, `.LegacyTabs`, etc. 

Additionally, anywhere that was applying css overrides in web was updated to support `Polaris-LegacyCard` wherever `Polaris-Card` styles were being overridden and once web was migrated, we moved `Polaris-Card` overrides. I recommend the same approach for LegacyFilters and LegacyTabs.